MoltenVK/MoltenVK/OS/MTLRenderPipelineDescriptor+MoltenVK.m

```diff
@@ -23,12 +23,16 @@
 @implementation MTLRenderPipelineDescriptor (MoltenVK)
 
 -(MTLPrimitiveTopologyClass) inputPrimitiveTopologyMVK {
+#if !MVK_TVOS
```

Comment:
In general, I'm not a fan of using the negative form `!MVK_TVOS`, `!MVK_IOS`, etc. I feel it makes it harder to expand to other platforms in the future because it hides the new platform as being highlighted as missing by the compiler during conditional compilation.

For instance, a bunch of existing `!MVK_IOS`'s would have made it harder to figure out where we needed `MVK_TVOS`'s.

Reply:
Got rid of these. I had to switch to selector syntax instead of property syntax to fix build on tvOS.